tests/unit/CMakeLists.txt

```diff
@@ -47,6 +47,9 @@ endif()
 
 # Don't run these tests if we are using MPI SERIAL.
 if (NOT PIO_USE_MPISERIAL)
+  add_executable (test_spmd EXCLUDE_FROM_ALL test_spmd.c test_common.c)
```

Comment:
Although these routines don't make a lot of sense with mpi serial they do work and should be tested.